tools/ci/platforms/windows.ps1

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env pswd
+############################################################################
+# tools/ci/platforms/windows.ps1
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+# Windows native
+
+# Set-PSDebug -Trace 0
+
+function add_path() {
+  param (
+    [string]$Path
+  )
+  if (!$Path) {
+    Write-Host "Error: add_path path file not found" -ForegroundColor Red
+    return
+  }
+  $envPaths = $env:Path -split ';'
+  if ($envPaths -notcontains $Path) {
+    Add-Content -Path "$NUTTXTOOLS\env.ps1" -Value "add_path $Path"
+    $env:PATH = "$Path;" + $env:PATH
+  }
+}
+
+function add_envpath {
+  param(
+    [string] $envfile
+  )
+  $head = @"
+#!/usr/bin/env pswd
+############################################################################
+#
+#

```

Comment:
Is this empty comment line meant to be doubled?

Reply:
added file name and description